File: Selection statement/Question2/question2.java
import java.util.Scanner;

public class question2 {
    public static void main(String[] args) {
        try (Scanner scan = new Scanner(System.in)) {
            int age;

            System.out.print("Enter your age: ");
            age = scan.nextInt();

            if (age >= 18) {
                System.out.println("You are an eligible voter.");
            } else {
                System.out.println("Please come back in " + (18 - age) + " years.");
            }
        }
    }
}

File: Selection statement/Question2/question2.cpp
#include <iostream>

int Eligiblity(int age)
{
    if (age >= 18)
    {
        std::cout << "Eligible to vote" << std::endl;
    }
    else
    {
        std::cout << "Please come back in " << 18 - age << "year" << std::endl;
    }
}

int main()
{
    int age;
    std::cout << "Enter your age: ";
    std::cin >> age;
    Eligiblity(age);
    return 0;
}